assets/js/components/KeyMetrics/MetricTileError.js

```diff
@@ -20,26 +20,47 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
+import { castArray } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import CTA from '../notifications/CTA';
 import ReportErrorActions from '../ReportErrorActions';
-import { __ } from '@wordpress/i18n';
+import { isInsufficientPermissionsError } from '../../util/errors';
 
 export default function MetricTileError( props ) {
 	const { error, headerText, moduleSlug } = props;
+	const hasInsufficientPermissionsError = castArray( error ).some(
+		isInsufficientPermissionsError
+	);
 
 	return (
 		<div className="googlesitekit-km-widget-tile--error">
 			<CTA
-				title={ __( 'Data loading failed', 'google-site-kit' ) }
+				title={
+					hasInsufficientPermissionsError
+						? __( 'Insufficient permissions', 'google-site-kit' )
+						: __( 'Data loading failed', 'google-site-kit' )
+				}
 				headerText={ headerText }
 				description=""
 				error
 			>
-				<ReportErrorActions moduleSlug={ moduleSlug } error={ error } />
+				<ReportErrorActions
+					moduleSlug={ moduleSlug }
+					error={ error }
+					getHelpText={
+						hasInsufficientPermissionsError
+							? 'Trouble getting access?'
```

Comment:
This is missing the `__()` translation function :)

Reply:
@techanvil, @10upsimon had the translation function here. However, since we had to translate the entire string, I've removed it from here and moved it to the `ReportErrorActions` component. Do we still need to translate here? Please see this [comment](https://github.com/google/site-kit-wp/pull/7554#discussion_r1319962318) and this [comment](https://github.com/google/site-kit-wp/pull/7554#discussion_r1319963486).

Reply:
Hi @hussain-t, the problem when trying to leave the translation to the point where the string is used in `ReportErrorActions` is that at that point we'd be trying to translate a variable string. So there's no clear translation at that point, it could be this text, or an empty string, or basically anything. Really, at the point where the translation is made in `ReportErrorActions`, all the translator knows is there's a string, a space, and a link. So the only thing they can really do there is to reorganise the order of these things, if say the language would benefit from having the link before the string instead of after it.

With that in mind, we still need to do the translation of the actual copy here, where the string literal is.

Reply:
This has now been updated - see https://github.com/google/site-kit-wp/commit/118d7f82a30c565a59f7e8b76d770179f4eec4eb.